src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -283,22 +287,32 @@ int64_t ClusterResourceScheduler::GetBestSchedulableNode(const TaskRequest &task
         // appropriate total resources in a timely manner. If there are
         // multiple feasible nodes, this algorithm can still introduce delays
         // because of inefficient load-balancing.
```

Comment:
I think you can remove the TODO now since you've introduced the better load balancing behavior.